src/app/grid/grid-action-strip/grid-action-strip-sample.ts

```diff
@@ -19,31 +17,27 @@ export class GridActionStripSampleComponent {
         this.data = DATA;
     }
 
-    public stateFormatter(value: string) {
-        return JSON.stringify(value);
-    }
-
-    public typeFormatter(value: string) {
-        return value.toUpperCase();
-    }
-
     public isDirty(rowContext: RowType) {
-        return rowContext && rowContext.deleted;
+        const isRowEdited = this.grid.transactions.getAggregatedChanges(true).find(x => x.id === rowContext?.key);
```

Comment:
Is it possible to use rowContext.inEditMode instead? Or because of batchEditing, you search in the aggregated changes?

Reply:
I see, inEditMode means currently in edit mode, while you look for an "edited" state. Feel free to log this as an enhacement issue for the library - to expose a similar state.